### ISSUE TITLE
use salmon rather than featurecounts

### ISSUE THREAD

**Issue by daler** [lcdb/lcdb-wf#271]
The DESeq2 vignette [recommends using salmon](http://bioconductor.org/packages/3.9/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#transcript-abundance-files-and-tximport-input) counts as input to DESeq2. We're already generating these as part of the workflow, so we should use them.

**Comment by hertafeldsr** [comment]
I was taking a look at this, and I noticed that the vignette says to use a tx2gene file (I think like the one that will be generated in #177) in the tximport() function, but the Rmd doesn't include one at the moment. @daler I think I remember you mentioning this and I wanted to double check that I should in fact include the tx2gene file. Also, should I replace featurecounts with salmon both in the initial exploratory dds and the downstream one for all the contrasts?

**Comment by daler** [comment]
Yep, we might need to wait for #177 to do this properly because we don't have one at the moment. Just to have something to work on, you can temporarily create one using an OrgDb or TxDb, or make one manually using the mappings that are already generated in the test data.

And as for featurecounts/salmon, the idea is to completely move away from featurecounts, so salmon will be used for both exploratory dds and the final one(s) used for DE.
